flink-core/src/main/java/org/apache/flink/configuration/ResourceManagerOptions.java

```diff
@@ -85,6 +85,20 @@
 		.defaultValue(-1L)
 		.withDescription("The timeout for a slot request to be discarded.");
 
+	/**
+	 * Time in milliseconds for the cluster initialization.
+	 * Used by the resource manager in standalone mode to decide in how long time the task executors should register.
+	 * After this time, slot requests that exceed capacity of all registered slots will be considered not satisfiable
+	 * and failed. If not configured, {@link JobManagerOptions#SLOT_REQUEST_TIMEOUT} will be used by default.
+	 */
+	public static final ConfigOption<Long> CLUSTER_INITIALIZATION_TIME = ConfigOptions
+		.key("resourcemanager.cluster-init-time")
```

Comment:
I feel that `cluster-init-time` seems to be too general. Do you think names like `resourcemanager.standalone.slot-resources-registration-timeout` would be better?